tests/scipy_signal_test.py

```diff
@@ -56,7 +56,7 @@
     ((65, 24), 24, 7, -2, -1),
 ]
 
-
+real_dtypes = jtu.dtypes.floating + jtu.dtypes.integer + jtu.dtypes.boolean
```

Comment:
nit: "real_dtypes" seems like a misleading name for a set that includes ints and bools.

Also, should we test complex types too?

Reply:
I just grabbed this from the `fft_tests`

we could test complex dtypes but the scipy (and this) routine raises an error for complex dtypes. see line 651. 

Should we try to assert the `ValueError`. If so, can you please share a code example of how we can catch that successfully?

Reply:
Oh, i didn’t notice that complex isn’t supported